prow/plugins/lgtm/lgtm_test.go

```diff
@@ -668,6 +668,8 @@ func TestHandlePullRequest(t *testing.T) {
 		issueComments map[int][]github.IssueComment
 
 		expectNoComments bool
+
+		stickyLgtm bool
```

Comment:
nit: It is helpful if the test case fields are formatted in two groups: the test conditions, and the expectations. Otherwise its a bit unclear if this is a test parameter or expected output.